### PR TITLE
libswiftnav 0.21

### PR DESCRIPTION
**Pull request by elfchief**

### CODE REVIEW

File: Formula/libswiftnav.rb [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make", "install"
+  end
+
+  test do
+    (testpath/"test.cpp").write <<-EOS.undent
```
Comment (dunn) [inline Formula/libswiftnav.rb]: I don't see anything being done with this test file?

**Comment by woodruffw** [comment]
Any update here @elfchief?

**Comment by elfchief** [comment]
Ah, right you are. Will fix.

**Comment by elfchief** [comment]
Ok, actual functioning test now. Sorry about that!

File: Formula/libswiftnav.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        crc = crc24q(test_data, 9, 0xB704CE);
+        if (crc != 0x21CF02) {
+          printf("libswiftnav CRC quick test failed: CRC of \\"123456789\\" with init value 0xB704CE should be 0x21CF02, not 0x%06X\\n", crc);
+          exit(1);
+        } else {
+          printf("libswiftnav CRC quick test successful, CRC = 0x21CF02\\n");
+          exit(0);
+        }
+      }
+EOS
```
Comment (MikeMcQuaid) [inline Formula/libswiftnav.rb]: Indent this by four spaces.

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format is `libswiftnav 0.21 (new formula)`. Please complete the issue template in future PRs where this format is detailed in the linked [Contributing guidelines](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md). Thanks!

**Pull request closed without merging**